### DISCUSSION TITLE
OS (Linux) provider

### DISCUSSION THREAD

**Discussion by alishavez7** [ClusterLabs/ha_cluster_exporter#181] [Q&A] [answered]
Hi Team,

I want to add the OS (Linux) provider but still there is documentation available on MS Doc site. Could you suggest what should be the value of "Node Exporter Endpoint" parameter value. I tried with the same Prometheus Endpoint url but it is not working.

Please suggest and also update the documentation how to add the OS(Linux) provider to Azure monitor for SAP Solution.

**Comment by stefanotorresi** [comment, marked as answer]
Hello @alishavez7!
I'm afraid you'll have to ask this in MS support channels, because this project by itself doesn't have any awareness of the MS Azure Monitor for SAP Solution.
We here serve merely as an open-source upstream for just one low-level component of the MS solution.
